infra/eligibility_screener/template/ecs.tf

```diff
@@ -3,15 +3,15 @@ resource "aws_security_group" "allow-screener-traffic" {
   name        = "allow_screener_traffic"
   description = "This rule blocks all traffic unless it is HTTPS for the eligibility screener"
   vpc_id      = module.constants.vpc_id
-
+  
   ingress {
-    description = "TCP traffic from VPC"
-    from_port   = 80
-    to_port     = 80
+    description = "Allow traffic from internet"
+    from_port   = 3000
+    to_port     = 3000
```

Comment:
Could we set the `to_port` to port 80 for now? That would mean that we could drop the `:3000` while we are still testing. We can switch to 443 once we get the url in place

Reply:
@aplybeah I think this is still not resolved?

Reply:
We talked through this and I misunderstood what the to and from ports represented here (the range of ports to be opened).